test/factories/submission.rb

```diff
@@ -0,0 +1,18 @@
+FactoryBot.define do
+  factory :user do
+    name "Oliver Sacks"
+    sequence(:email) { |n| "olivers#{n}@email.com" }
+    password "secret"
+  end
```

Comment:
So this factory file is named `submission`, but it's including a definition for a `user`. I suggest moving this definition to a `test/factories/user.rb` file.

Reply:
I thought about that before. Would you advise against one main `factories/factories.rb` file?